modules/sc-mesh-secure-deployment/src/2_0/features/PHY/PHY_CRA_scripts/SP_CRA_v7.py

```diff
@@ -0,0 +1,214 @@
+import socket
+import threading
+import random
+import pickle
+from mat4py import loadmat
+import time
+import logging
+import numpy as np
+import os
+from getmac import get_mac_address
+
+class PHYCRA:
+    def __init__(self):
+        logging.basicConfig(level=logging.INFO, filename='server_log.txt', filemode='w',
```

Comment:
Log file location will be current working directory.  Maybe better place is to store /tmp/ or /var/run/log/.

Reply:
Done